src/lucky/cookies/flash_store.cr

```diff
@@ -0,0 +1,53 @@
+class Lucky::FlashStore
+  SESSION_KEY = "_flash"
+  alias Key = String | Symbol
+
+  delegate each, to: all
+
+  @now = {} of String => String
+  @next = {} of String => String
+
+  def self.from_session(session : Lucky::SessionCookie) : Lucky::FlashStore
+    new.from_session(session)
+  end
+
+  def from_session(session : Lucky::SessionCookie) : Lucky::FlashStore
+    json = session.get?(SESSION_KEY) || "{}"
```

Comment:
```cr
session.get?(SESSION_KEY).try do |json|
  JSON.parse(json).as_h.each do |key, value|
    @now[key.to_s] = value.to_s
  end
end
```

This will avoid parsing an empty object (`"{}"`).